### PR TITLE
Link the Download and Install Section of the README File to B.io

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This updates the "Download and Install" section to point to [1].

[1] https://ballerina.io/learn/installing-ballerina/

Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/214

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,40 +28,9 @@ You can use one of the following options to try out Ballerina.
...
-
-Alternatively, you can install Ballerina from the source using the following instructions.
-
-#### Prerequisites
-
-* [Oracle JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) or [OpenJDK 8](http://openjdk.java.net/install/)
-* [Node (v8.9.x) + npm (v5.6.0 or later)](https://nodejs.org/en/download/)
-* [Docker](https://www.docker.com/get-docker)
-
-#### Building the source
```
Comment (anupama-pathirage) [inline README.md]: Why we are removing installing from the source from Readme file? That information is not there in https://ballerina.io/learn/installing-ballerina/. So from where users can get this details?

Comment (praneesha) [reply README.md]: This PR is to add that info to the "Installing Ballerina" page.

https://github.com/ballerina-platform/ballerina-dev-website/pull/215

Comment (anupama-pathirage) [reply README.md]: That information is already there in https://github.com/ballerina-platform/ballerina-dev-website/blob/master/contribution-guide.md with more details. So with this PR are we going to duplicate the same information in both places. IMO, it is not correct. The content in the both places are having different information even right now. If we really want to have it in web site then we need to have a fully descriptive separate page on building from source. (see go for example https://golang.org/doc/install/source)

Comment (praneesha) [reply README.md]: +1 will merge the info and create a separate page.

Comment (praneesha) [reply README.md]: IMHO, we do not have content to add a separate page for the topic "Installing from Source ATM like in Go. Therefore, merged the content that was there in the Contribution Guide to the "Installing from source" section in the "Installing Ballerina" page via ballerina-platform/ballerina-dev-website#215.

Now, both the Contribution Guide in B.io and README file in the ballerina-lang repo links to the "Installing from source" section in the "Installing Ballerina" via ballerina-platform/ballerina-dev-website#215.

**Pull request merged**